### ISSUE TITLE
cs_accessibility.py: On-screen keyboard - changing screen size/position doesn't take into effect until keyboard is fully reloaded

### ISSUE THREAD

**Issue by ItzSwirlz** [linuxmint/cinnamon#9793]
```
 * Cinnamon version (cinnamon --version) 4.8.3
 * Mint 20.1 Beta
 * Graphics hardware *and* driver used - GNOME Boxes
 *  64 bit
 ```

**Issue**
When changing the position of the on-screen-keyboard and the size of the keyboard doesn't change until the on-screen-keyboard is fully reloaded (disabling and reenabling)

**Steps to reproduce**
Enable on-screen keyboard
Change position on screen (bottom, top, etc) and
Change the size of the on-screen-keyboard

You will see nothing changes until you:
Disable and renable the on-screen keyboard

**Expected behaviour**
The On-screen keyboard updates.

**Other information**
I would PR if I could figure out where the problem is but I can't.

**Configuration info, if applicable**
Mintreport doesn't show anything.

[xsession-errors.txt](https://github.com/linuxmint/cinnamon/files/5742327/xsession-errors.txt)
[syslog.txt](https://github.com/linuxmint/cinnamon/files/5742331/syslog.txt)

**Comment by ItzSwirlz** [comment]
It seems that all the settings are attached correctly; it's either a problem with cs_accessibility.py or I think this should be relocated to cinnamon-settings-daemon under the keyboard plugin.

**Comment by ItzSwirlz** [comment]
Closing as a duplicate of an issue I already made.
